learnosity_sdk/request/init.py

```diff
@@ -93,11 +101,28 @@ def generate(self, encode=True):
             if self.action is not None:
                 output['action'] = self.action
 
+        if self.__telemetry_enabled:
+            output['meta'] = self.get_meta()
```

Comment:
We don't account for it in PHP, but in Node and Java I've made sure that we _edit or add_ the `meta` object, instead of completely replacing it. I'm not sure if this is particularly important but might be useful to do here in case one of our APIs starts needing a `meta` field in any of its requests.

https://github.com/Learnosity/learnosity-sdk-nodejs/blob/master/index.js#L41-L51
https://github.com/Learnosity/learnosity-sdk-java/blob/master/src/main/java/learnositysdk/request/Telemetry.java#L56-L63

Reply:
Seems a bit yucky to prejudge the actions of a future developer by checking for a condition that definitely isn't true at the moment, no? Output is created as an empty dict ~40 lines earlier and stuff is added to it as needed.

Reply:
Actually, now that you point that out - the `meta` is supposed to be added to the actual `request` object. Not the root level object for the request.

So my point here is that a user could add a `meta` field to their request data (again, possible in the case that one of our many APIs starts using that field) and this would overwrite that data which the user is trying to send to our sever.

Reply:
Ah, that makes sense, thanks @flakeparadigm.